.github/workflows/pr_compliance.yml

```diff
@@ -0,0 +1,21 @@
+name: validate pr
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+    branches:
+      - master
+
+jobs:
+  validate-pr:
+    runs-on: ubuntu-latest
+    env:
+      REQUEST_BODY: ${{ github.event.pull_request.body }}
+      REQUEST_TITLE: ${{ github.event.pull_request.title }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+      - name: run script
+        run: python scripts/pr_compliance.py > test.log || { echo "::error::pr_compliance.py $(cat test.log)" && exit 1; }
```

Comment:
i don't think we need a separate flow. we just need the job runs in parallel to the `build` job in `bot.yml`. When people create a PR and not compliant in the description, the testing jobs still run. they can just fix the description and re-run the validate-pr job to pass the CI.